ssv/committee_runner.go

```diff
@@ -236,12 +234,12 @@ func (cr CommitteeRunner) ProcessPostConsensus(signedMsg *types.PartialSignature
 
 			// Get the beacon object related to root
 			validatorObjs, exists := beaconObjects[validator]
-            if !exists {
+			if !exists {
 				anyErr = errors.Wrap(err, "could not find beacon object for validator")
 				continue
 			}
 			sszObject, rexists := validatorObjs[root]
-            if !rexists {
+			if !rexists {
```

Comment:
Pushed a change. Is that what you meant?

Reply:
not exactly...
if you do it on a one line with `;` you will also limit the scope of `exists` to the `if` block...
Currently I think (not sure) you are overshadowing the previous `exists` and this may cause subtle bugs